### ISSUE TITLE
JSON parser/builder nodes

### ISSUE THREAD

**Issue by Zetaphor** [Yellow-Dog-Man/Resonite-Issues#946]
### Is your feature request related to a problem? Please describe.

There is robust support in Resonite for HTTP requests and Websocket connections. However handling responses to either of these becomes a chore in tedium when you have to resort to parsing JSON (the most common response format) using the various string parsing nodes. Conversely, building JSON payloads to submit over HTTP or Websocket is also difficult, though admittedly less painful thanks to the string builder node.

### Describe the solution you'd like

A set of nodes that allow the user to parse, build, and validate JSON strings.

### Describe alternatives you've considered

In the past I've had to resort to overly complex string parsing to decipher JSON responses, and extensive use of the string formatter node to build JSON payloads.

**Comment by Banane9** [comment]
@ukilop has made a "universal" JSON parser in protoflux, but integrated support is only planned after #572

**Comment by Frooxius** [comment]
Yes, generic collection support is a prerequisite for this. However once the collections are added, JSON nodes are going to be pretty quick to add in.

**Comment by epicEaston197** [comment]
Now that we have collections Should we be expecting JSON nodes to come relatively soon?
